spi/core-spi/src/main/java/org/eclipse/dataspaceconnector/spi/iam/TokenValidationService.java

```diff
@@ -16,11 +16,19 @@
 
 import org.eclipse.dataspaceconnector.spi.result.Result;
 import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
 
 /**
  * Interface for validating token.
  */
 @FunctionalInterface
 public interface TokenValidationService {
-    Result<ClaimToken> validate(@NotNull String token);
+    Result<ClaimToken> validate(@NotNull String token, @Nullable Map<String, Object> additional);
```

Comment:
Please document methods

Reply:
Done (please check again if ok) - in general the code documentation should be expanded